packages/schemas/alterations/next-1664356000-add-created-at-column-to-users.ts

```diff
@@ -0,0 +1,20 @@
+import { sql } from 'slonik';
+
+import { AlterationScript } from '../lib/types/alteration';
+
+const alteration: AlterationScript = {
+  up: async (pool) => {
+    await pool.query(sql`
+      alter table users add column created_at timestamptz not null default (now());
```

Comment:
whenever run the db migrations, all the existing users will become new users? Shouldn't we load old users data from the log?

Reply:
#2031 Done